examples/test1/build.gradle

```diff
@@ -1,67 +1,49 @@
 /*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Based on jruyi/thrift-gradle-plugin/../examples/test1/build.gradle
+ * Modified 2023 LINE Corporation
```

Comment:
Just noticed that I have put both LINE/Apache 2 copyright header together in previous PR... Should I change the [previous file](https://github.com/line/thrift-gradle-plugin/blob/master/src/main/java/org/jruyi/gradle/thrift/plugin/CompileThrift.java#L1-L28) to follow the convention here?

Reply:
Maybe just leave as is and later we can unify to the same format.